src/ring/mock/request.clj

```diff
@@ -113,11 +115,18 @@
                         "application/octet-stream")))
         filename (or (:filename param)
                      (when (file? value) (.getName ^File value)))]
-    (.addBinaryBody builder (name k) value mimetype filename)))
+    
+    (cond
+      (bytes? value) (.addBinaryBody builder ^String (name k) ^"[B" value ^ContentType mimetype ^String filename)
+      (file? value) (.addBinaryBody builder ^String (name k) ^File value ^ContentType mimetype ^String filename)
+      (instance? InputStream value) (.addBinaryBody builder ^String (name k) ^InputStream value ^ContentType mimetype ^String filename)
+      :else
+      builder
+      )))
```

Comment:
Trailing brackets should not be on their own line.

Reply:
Done